langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css

```diff
@@ -571,4 +587,4 @@ div.block div.block span.interfaceName {
 
 div.contentContainer ul.blockList li.blockList h2{
     padding-bottom:0px;
-}
+}
```

Comment:
Please revert this unnecessary change.

Reply:
Only line numbers are changed here right, I couldn't understand what needs to be reverted here. Please suggest, Thank you.

Reply:
I believe what Keith is asking is to add a newline at the end of line 590 so we don't see that difference from the original code.

Reply:
Thanks for the clarification.